modules/xmpp/xmpp.js

```diff
@@ -489,6 +489,20 @@ export default class XMPP extends Listenable {
 
         this.sendDiscoInfo = true;
 
+        if (this.connection._stropheConn && this.connection._stropheConn._addSysHandler) {
+            this.connection._stropheConn._addSysHandler(msg => {
+                this.sendDiscoInfo = false;
```

Comment:
Is this a one-time handler? If the XMPP server does not send the expected message, is this going to fire with the next regular `message` stanza we receive?

Reply:
this will fire anytime for messages with type conference-info.

Reply:
Ahh, I missed the other arguments. What makes a message have a type of "conference-info"? It contains a "<conference-info" element?

Nit: we haven't joined a conference yet, is for example "service-info" a more accurate name?

Reply:
https://github.com/jitsi/jitsi-meet/pull/9052/commits/1840d4d0bc4bd468c98ee9d7b52d41acba8cd334#diff-51cb31fcf1135ed84b83d4324d1086d258898f786d53cecf56c3d8ab85e07413R28
We set it, ok. Make sense, I will change it.

Reply:
Fixed.